docs/articles/configure-win-e.md

```diff
@@ -15,4 +18,14 @@ FilesLocation := USERPROFILE . "\AppData\Local\Microsoft\WindowsApps\files.exe"
         WinActivate % FilesTitle     ; Set focus
     }
 return
-```
\ No newline at end of file
+```
+
+**Option 2: modifying the registry**
+
+*This method involves modifying the registry, make sure to create a backup beforehand and proceed at your own risk. Please keep in mind that this method is unsupported and may not work for everyone.*
+
+You can also configure Files to launch using the Win + E shortcut without an external program:
+1. Create a backup of the registry, make sure to store the backup in your desktop folder so that you can access it in the event that Files won't open.
+2. Download *both* these .reg files [1](https://github.com/files-community/files-community.github.io/data/UndoOpenFilesOnWinE.reg) and [2](https://github.com/files-community/files-community.github.io/data/OpenFilesOnWinE.reg) and place them *on the desktop*
+3. Run `OpenFilesOnWinE.reg` to open Files on Win+E
+4. Run `UndoOpenFilesOnWinE.reg` to restore windows explorer
```

Comment:
Should we add instructions for manually editing the registry?

Reply:
Done 👍